### PR TITLE
Display identity creation error details

### PR DESCRIPTION
**Pull request by orhoj**
## Purpose
Notabene now provides error message details in the `detail` field. This PR ensures that we save that information on rejected identities, and that we display the message to the user.

## Changes
- `detail` is saved in the identity table if the identity creation fails.
- the `detail` field is displayed for failed identities.

## Checklist
- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by soerenbf — CHANGES_REQUESTED** [review]
As agreed, I pushed some changes to the presentation on the `IdentityCard`.

If an account is rejected, I get an error about checking the node status, as the account address isn't on chain. Should be able to avoid this?

Screenshot 2021-11-19 at 11 49 48

**Comment by soerenbf** [comment]
Could also link this PR to #165

File: app/components/FailedIdentityModal.tsx [hunk 1]
```diff
@@ -35,11 +38,29 @@ export default function FailedIdentityModal({
         }
     }, [isRejected, setModalOpen]);
 
+    const decriptionComponent = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (orhoj) [reply app/components/FailedIdentityModal.tsx:41]: I renamed it to `description`.

**Comment by orhoj** [comment]
> As agreed, I pushed some changes to the presentation on the `IdentityCard`.
> 
> If an account is rejected, I get an error about checking the node status, as the account address isn't on chain. Should be able to avoid this?
> 
> Screenshot 2021-11-19 at 11 49 48

Yeah. That should not be happening, but it may be related to the node-sdk changes Søren made. It is, however, unrelated to this PR.

**Comment by soerenbf** [comment]
> > As agreed, I pushed some changes to the presentation on the `IdentityCard`.
> > If an account is rejected, I get an error about checking the node status, as the account address isn't on chain. Should be able to avoid this?

> Yeah. That should not be happening, but it may be related to the node-sdk changes Søren made. It is, however, unrelated to this PR.

I'll file a bug instead.

**Review by soerenbf — APPROVED** [review]

**Pull request merged**